public/data/feedData.json

```
@@ -0,0 +1,50 @@
+[
+  {
+    "feedId": 0,
```

```suggestion
    "feedId": 1,
```
id가 0 부터 시작하는건 좀 어색하네요!